CHANGELOG.md

```diff
@@ -1,7 +1,30 @@
 # Changelog
 
 ## [0.12.1](https://github.com/brave/browser-laptop/releases/v0.12.1dev)
-- (todo)
+ - Added Malay (Malaysia) language. ([#3843](https://github.com/brave/browser-laptop/issues/3843))
+ - Added extensions. ([#3926](https://github.com/brave/browser-laptop/issues/3926))([#3867](https://github.com/brave/browser-laptop/issues/3867))([#2224](https://github.com/brave/browser-laptop/issues/2224))
```

Comment:
This is misleading and makes the reader think we have new extensions that are added, which is not the case, if it was we woul say which ones.
The first one is an about:extensions page
The last 2 are the same but should be more description, something like Added support for extension action buttons for password managers.

I'd put the Malay language at the bottom of the added list since it isn't applicable to everyone.